src/steps/generic.rs

```diff
@@ -369,15 +369,33 @@ pub fn run_gcloud_components_update(ctx: &ExecutionContext) -> Result<()> {
     let gcloud = require("gcloud")?;
 
     if gcloud.starts_with("/snap") {
-        Ok(())
-    } else {
-        print_separator("gcloud");
+        return Ok(());
+    }
 
-        ctx.run_type()
-            .execute(gcloud)
-            .args(["components", "update", "--quiet"])
-            .status_checked()
+    print_separator("gcloud");
+
+    let output = ctx
+        .run_type()
+        .execute(&gcloud)
+        .args(["components", "update", "--quiet"])
+        .output()?;
```

Comment:
```suggestion
        .output_utf8()?;
```
That way you can remove the `String::from_utf8` call